.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,24 @@
-[comment]: # (Please replace ... with your information. Remove < and >)
 [comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
-
-If you submit PR for issue with bounty then write here Fixes #NN where NN is issue number
-
-*otherwise*
-
 fixes #...
 
 ### Summary
-
 [comment]: # (Summarise the problem and how the pull request solves it)
 ...
 
-<!-- (Optional, remove if no changes to documentation) -->
-Documentation change PR (review please): https://github.com/status-im/status.im/pull/xxx
-
 ### Review notes
-<!-- (Optional. Specify if something in particular should be looked at, or ignored, during review) -->
+[comment]: # (Optional. Specify if something in particular should be looked at, or ignored, during review)
 
 ### Testing notes
```

Comment:
I usually raise the level on up. That makes the headline in GH have an underline and stand out better, also gives the person a chance to use multiple level-3 headlines if desired. Personal preference, but I'd vote for raising all headlines one level up.

Reply:
@ilmotta do you mean having `## Review notes` instead of `### Review notes`?